hack/eks

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: terraform enviroment variables need to be set
+INST_PROFILE_NAME=suwang-eks-worker-nodes-1-NodeInstanceProfile-8LL239W4W3GN
```

Comment:
Please add a check instead to make sure this environment variable is set properly, and print an error message otherwise.